utils/generic.py

```diff
@@ -608,8 +634,8 @@ def get_all_dups(s, dups, samples_duplicates):
     return duplicated_samples
 
 
-def infer_families(kin_ht: hl.Table,
-                   sex: Dict[str, bool],
+def infer_families(relatedness_ht: hl.Table,
+                   sex: Union[hl.Table, Dict[str, bool]],
```

Comment:
the second value of second_degree_threshold is never used downstream so can be removed right?